metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -0,0 +1,26 @@
+=== windows perfmon MetricSet
+
+This is the perfmon metricset of the module windows.
+
+[float]
+=== Features and configuration
+
+Metricset to collect performance counters.
+Example configuration:
+```
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 10s
+  perfmon.counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
```

Comment:
That is definitively an issue with my suggestion. The benefit is that the user as the full flexibility on where to put events and groups are not strictly required. Also there is less indentation yaml mess that can happen.

If it is natural to have a group for the events and users kind of expect that in the result your approach definitively feels more natural. 

@andrewkroh Any thoughts?